schema/tables/puppet_facts.yml

```diff
@@ -1,29 +0,0 @@
-name: puppet_facts
```

Comment:
We'll be readding this on a separate PR.

This is not present in fleetd because we haven't upgraded https://github.com/macadmins/osquery-extension yet.